src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,50 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = fileReader(fromFileName);
+        fileWriter(toFileName, report);
+    }
+
+    private void fileWriter(String fileToWrite, String dataToBeWritten) {
+        File resultedInfo = new File(fileToWrite);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(resultedInfo))) {
+            writer.write(dataToBeWritten);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file", e);
+        }
+    }
 
+    private String fileReader(String incomeFile) {
```

Comment:
reader should only read from file, calculations or report creation are other actions